source/db/protocols/eo-protocols.json

```diff
@@ -18,6 +18,15 @@
     }]
   },
   {
+    "name": "Matrix",
```

Comment:
It was missing and the file was to short for my script to put OMEMO in the correct place.